README.md

```diff
@@ -252,11 +280,7 @@ gmusic.on('change:playback', function (mode) {
 });
 ```
 
-- mode `String` - Phase that a song is in (e.g. 0, 1, 2)
-    - 0 - Song is stopped
-    - 1 - Song is paused
-    - 2 - Song is playing
-    - Values are available via `GMusic.Playback.STOPPED`, `GMusic.Playback.PAUSED`, and `GMusic.Playback.PLAYING`
+- mode `String` - Same as return value from `playback.getPlaybackState()`
```

Comment:
We should use the same phrasing between the "return value" notes. Standardize on either "of" or "from", I think either will work